portingdb/templates/howto.html

```diff
@@ -33,13 +33,15 @@ <h2 id="package-it">Package it</h2>
         </p>
         <h3>What's What</h3>
         <p>
-            Look further down the sidebar for the headline <i>RPMs</i>. There you will see all the RPM (sub)packages contained in this package. Sometimes there's just the base package, sometimes it has several subpackages. What we want to know is if there is a (sub)package for Python 3. If there is only a Python 2 (sub)package, then we know that this package needs to be ported.
+            On the package page in PortingDB, look down the sidebar for the headline <i>RPMs</i>. There you will see all the RPM (sub)packages contained in this package. Sometimes there's just the base package, sometimes it has several subpackages. What we want to know is if there is a (sub)package for Python 3. If there is only a Python 2 (sub)package, then we know that this package needs to be ported.
         </p><p>
             However, sometimes you will find a package that is still mispackaged even though it already has a Python 3 (sub)package. In those cases, something isn't quite right. Usually the Python 3 subpackage depends on both Python 3 <i>and</i> Python 2. That is unfortunate, because it basically renders the porting effort null.
         </p>
         <h3>Make a Patch</h3>
         <p>
-            In either case, look a bit further down still and find a link to the associated Bugzilla report (under <i>Links→bug</i> in the sidebar). Go there and join the discussion. Find out what's the status quo and if anyone is working on it. If not, be the one that submits a patch!
+            In either case, look a bit further down on the package page in PortingDB and find a link to the associated Bugzilla report (under <i>Links→bug</i> in the sidebar). Go there and join the discussion. Find out what's the status quo and if anyone is working on it. If not, be the one that submits a patch!
+        </p><p>
+            If you don't find a link to Bugzilla, you'll need to investigate the issue and <a href="https://fedoraproject.org/wiki/User:Pviktori/Python_3_Bug_Filing">file a bug</a> yourself. A hint will usually provided just under the Fedora heading on the package page. When filing a bug, please make it block the <b>PYTHON3</b> tracking bug.
```

Comment:
Good addition. Though I would omit the:

> When filing a bug, please make it block the PYTHON3 tracking bug.

That is best left for the file-a-bug wiki page, which I am planning to give a bit of a polish too.